index.html

```diff
@@ -6,58 +6,65 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Cabin|Roboto:400,700|Roboto+Condensed:400,700" rel="stylesheet">
+
 </head>
 
 <body>
-    <header style="text-align: center;">
+    <header>
         <h1>Timur Buchberger</h1>
         <nav>
-            <a href="#contact">Contact</a>
             <a href="#bio">Bio</a>
+            <a href="#contact">Contact</a>
         </nav>
     </header>
-    <figure>
-        <img src="img/Cat.jpeg"
-        alt="placeholder picture of a cute cat">
-        <br />
+    <div class="hero">Responsive Front End Development</div>
+    <!--<figure>
+        <img src="img/Cat.jpeg" alt="placeholder picture of a cute cat">
         <figcaption>A placeholder picture of a beautiful cat</figcaption>
-    </figure>
-    <section>
+    </figure>-->
+    <section class="interests">
         <h2>Interests</h2>
         <ul>
-            <li>Programming</li>
-            <li>Good Food</li>
-            <li>Traveling</li>
-            <li>Tennis</li>
-            <li>History</li>
+            <li class="card"><img src="icons/laptop.svg">Programming</li>
+            <li class="card"><img src="icons/food.svg">Good Food</li>
+            <li class="card"><img src="icons/vacay.svg">Traveling</li>
+            <li class="card"><img src="icons/tennis.svg">Tennis</li>
+            <li class="card"><img src="icons/books.svg">History</li>
```

Comment:
The `alt` attribute is missing in these images!

Reply:
Thanks, will add them immediately.